examples/block_jid.py

```diff
@@ -114,6 +116,11 @@ def jid(s):
             help="If given, prints the block list at the end of the operation",
         )
 
+        config_default = os.path.expanduser(
+            '~/.config/aioxmpp_credentials.ini')
```

Comment:
In general I agree. I don’t think it makes too much sense for these examples though, because they are not installed with neither the pip nor the AUR package. Their intended audience is not end users, but developers who want to take a look on how to use the aioxmpp API.

I can see some advantages when one has multiple aioxmpp repositories checked out, which is handy while preparing releases.

As a middle way, let’s go with:

```
os.path.join(os.environ.get("XDG_CONFIG_HOME", os.path.expanduser("~/.config")), "aioxmpp_examples.ini")
```

This will work correctly on XDG-compliant distributions, and everyone else gets a more-or-less usable default. That doesn’t hurt, because before there was no default at all, the file is unlikely to exist or contain any surprising content, and one can still override that using the command line option.